README.md

```diff
@@ -476,38 +470,38 @@ the backups and store them at `/mnt/backup/dashboard`, so we should have two
 copies at all times.
 
 ## Code Tips
-This is just a collection of small things to be aware of if you're going to write 
-code for the dashboard. To help with profiling and debugging consider using the 
+This is just a collection of small things to be aware of if you're going to write
+code for the dashboard. To help with profiling and debugging consider using the
```

Comment:
```suggestion
code for the dashboard. To help with profiling and debugging, consider using the
```

Reply:
Maybe eventually, we can create a `CONTRIBUTING.md` file and move this content there.

Reply:
Yeah definitely!